korge/src/commonMain/kotlin/com/soywiz/korge/view/View.kt

```diff
@@ -1468,12 +1466,10 @@ fun <T : View> T.onNextFrame(updatable: T.(views: Views) -> Unit) {
 // @TODO: This should be computed and invalidated when a view is attached to a container
 val View?.ancestorCount: Int get() {
     var count = 0
-    var node = this
-    while (node != null) {
-        node = node.parent
-        if (node != null) {
-            count++
-        }
+    var parent = this?.parent
+    while (parent != null) {
+        count++
+        parent = parent.parent
     }
     return count
```

Comment:
Risky, but I think this is correct. Also there are tests covering this, so I guess this is ok